oscarlab/graphene
# [Docs] Clean-up, unify and rearrange Graphene documentation

## Description of the changes 

Our documentation at https://graphene.readthedocs.io/ doesn't look good. This is the first step at visually improving it.

## How to test this PR? 

Check out the documentation.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/oscarlab/graphene/1742)
